web_src/less/_base.less

```diff
@@ -1371,6 +1433,52 @@ i.icon.centerlock {
   color: var(--color-text-light);
 }
 
+.ui.active.label {
+  background: var(--color-secondary);
+  border-color: var(--color-secondary);
+  color: var(--color-text-dark);
+}
+
+.ui.active.label:hover,
+a.ui.active.label:hover {
+  background: var(--color-secondary-dark-1);
+  color: var(--color-text-dark);
+}
+
+.ui.form .field.error .ui.multiple.selection.dropdown > .label {
+  color: var(--color-dropdown-error-text);
+  background: var(--color-dropdown-error-bg);
+
+  &:hover {
+    background: var(--color-dropdown-error-hover);
+  }
+
+  &.active {
+    background: var(--color-dropdown-error-active);
+
+    &:hover {
+      background: var(--color-dropdown-error-active-hover);
+    }
+  }
+
+  &.red {
+    color: var(--color-dropdown-error-red-text);
+    background: var(--color-dropdown-error-red-bg);
+
+    &:hover {
+      background: var(--color-dropdown-error-red-hover);
+    }
+
+    &.active {
+      background: var(--color-dropdown-error-red-active);
+
+      &:hover {
+        background: var(--color-dropdown-error-red-active-hover);
+      }
+    }
+  }
+}
+
```

Comment:
Can we maybe not use nested syntax to override fomantic? I prefer plain selectors in this case because it's easiert to search exact selector strings in fomantic source and just copy from there. Nested syntax would break this searching.